uwrt_utils/include/uwrt_utils/uwrt_can.h

```diff
@@ -0,0 +1,48 @@
+#include <string>
+#include<vector>
+#include<map>
+#include <thread>
+#include <mutex>
+#include <chrono>
+#include <linux/can.h>
+#include <linux/can/raw.h>
+#include <net/if.h>
+
+namespace uwrt_can {
+
+class UWRTCANWrapper {
+ public:
+  explicit UWRTCANWrapper(const std::string& name, const std::string& device, int delay_time = 1) 
```

Comment:
should probably have an optional second can interface name for the failover interface. this is used for all can nodes, except roboteq, so there needs to be a way to only specify one if name too.

also nit: interface or interface_name or can_interface_name instead of device.